erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md

```diff
@@ -0,0 +1,64 @@
+# Promotional Scheme
+
+Promotional schemes are the key to successful retail stores. Promotional schemes can be easily configure in the erpnext. Promotional scheme is linked to the pricing rule, against each slab system will generate the pricing rule.
+
+# 1. How to create a Promotional Scheme
+
+1. Go to: **Selling > Items and Pricing > Promotional Scheme > New**.
+1. Set a title for the rule.
+1. Select what to Apply On like Item Code, Item Group, Brand etc.
+1. Based on the apply on system will give you the provision to select the Item Code / Item Group / Brand in the table.
+1. Then configured the party information.
+1. Set promotional schemes for price dicsount, product discount.
+1. User can also apply the discount on the other item / item group / brand by selcting the value for Apply Rule On Other field.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes.png">
+
+## 1.1 Price Discount
+
+In this type of promotional schemes, the user gets an option to set the discount in terms of percentage or amount based on the min quantity, max quantity, min amount and max amount on the products. User can also configure the scheme where they can set the flat rate for the product based on the quantity or the amount of the product.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-price-discount.png">
+
+## 1.2 Product Discount
+
+In this type of promotional schemes, the user gets an option to give the free product on purchase of the same or different product with conditions like min quantity, max quantity, min amount, max amount.
+
+<img alt="Promotional Scheme" class="screenshot" src="{{docs_base_url}}/assets/img/promotional-schemes/promotional-schemes-product-discount.png">
+
+# 2. How to configure a Promotional Scheme
+
+Lets understand how to configure a promotional scheme in erpnext using some examples.
+
+## 2.1 Mixed Conditions Schemes
+
+Customer A has purchased 10 quantities of Britannia Cake 5 Rs packet and 5 quantities of Britannia Cake 10 Rs packet. Now shopkeeper wants to give the discount of 10% to the customer A. This shopkeeper also wants to give 10 % discount to the customer B who has purchased 15 quantities of Britannia Cake 5 Rs packet. So basically user wants to apply the discount on the products Britannia Cake 5 Rs, Britannia Cake 10 Rs only if his clients have purchased the 15 quantities of any product or sum of both product.
```

Comment:
customer A,,,,,, customer B (remove 'the')
remove the: the products Britannia Cake 5 Rs
remove the before 15 quantities